reports/management/commands/generate_csv.py

```diff
@@ -0,0 +1,23 @@
+from django.core.management.base import BaseCommand
+from reports.report_builder import XLSXReportBuilder
+
+from reports.forms import GlobalForm,
```

Comment:
Lets call the command `generate-dataset`... csv is just the file format we'll use but it's possible to generate it in even Excel.

```suggestion
from reports.forms import GlobalForm
```